common/src/main/res/values/styles.xml

```diff
@@ -767,6 +791,17 @@
         <item name="android:textColor">@color/gray_400</item>
     </style>
 
+    <style name="Caption.Gray400.13">
```

Comment:
Caption has text size 12 per our [design system](https://www.figma.com/file/azdJACb5WmivxYVhB5q46F/Android-%7C-Design-system?node-id=479%3A147). Let's not add confusion to the typography styles. 13 text is probably a button caption.